src/components/Dropdown/Dropdown.jsx

```diff
@@ -253,10 +253,10 @@ const Dropdown = ({
         Input,
         Option,
         Control,
-        ...(finalValueRenderer && { SingleValue }),
+        SingleValue,
```

Comment:
we now set a single value component also when we did not receive a custom renderer (for adding an avatar or icon)